Kernel/ACPI/ACPIStaticParser.h

```diff
@@ -39,8 +39,8 @@ namespace ACPI {
         static bool is_initialized();
 
         virtual PhysicalAddress find_table(const char* sig) override;
-        virtual void do_acpi_reboot() override;
-        virtual void do_acpi_shutdown() override;
+        virtual int do_acpi_reboot() override;
+        virtual int do_acpi_shutdown() override;
```

Comment:
I don't understand the purpose of `int` here. If you only want to signal success/failure, please use `bool`.
If you want to return a POSIX error code, please use `KResult` :)